src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -56,6 +59,29 @@ public class IPAddress
         //   1: Terminating null byte
         internal const int NumberOfLabels = IPAddressParserStatics.IPv6AddressBytes / 2;
 
+        // When IPv6 support was added to the .NET Framework, the public Address property was marked as Obsolete.
+        // The public obsolete Address property has not been carried forward in .NET Core, but remains here as
+        // internal so internal types that understand IPv4 can still access it without obsolete warnings.
+        internal long Address
+        {
+            get { return _addressOrScopeId; }
+        }
+
+        internal string StringRepresentation
```

Comment:
I couldn't find any internal callers of `StringRepresentation` in corefx, but added the property just in case there is other code that isn't open source yet that needs to call this member. Otherwise, this property can go away. (Even if there are internal callers that need it; I'd wonder why they couldn't be updated to call `ToString()` instead)